src/test/java/org/reaktivity/specification/amqp/streams/LinkIT.java

```diff
@@ -84,4 +84,16 @@ public void shouldReceiveTransferAndSendDisposition() throws Exception
         k3po.notifyBarrier("ROUTED_SERVER");
         k3po.finish();
     }
+
+    @Test
+    @Specification({
+        "${scripts}/transfer.as.sender/client",
+        "${scripts}/transfer.as.sender/server"})
```

Comment:
Suggest renaming `transfer.as.sender` to `transfer.to.server`.
Also rename method to `shouldTransferToServer`